tooling/backend_interface/src/proof_system.rs

```diff
@@ -36,6 +36,18 @@ impl Backend {
         InfoCommand { crs_path: self.crs_directory() }.run(binary_path)
     }
 
+    /// If we cannot get a valid backend, returns the default backend which supports all the opcodes
+    /// and uses Plonk with width 3
+    /// The function also print a message saying we could not find a backend
```

Comment:
```suggestion
    /// The function also prints a message saying we could not find a backend
```
Should we maybe also have the message state that we are using the default PLONK width 3 backend that supports all the opcodes

Reply:
Done, I have also updated the message